app/lib/pdf_filler/nc_d400_pdf.rb

```diff
@@ -38,6 +38,7 @@ def hash_for_pdf
         y_d400wf_sv1yes: checkbox_value(@submission.data_source.spouse_veteran_yes?),
         y_d400wf_sv1no: checkbox_value(@submission.data_source.spouse_veteran_no?),
         y_d400wf_rs2yes: @submission.data_source.filing_status_mfj? ? 'Yes' : 'Off',
+        y_d400wf_fedex1no: 'Yes',
```

Comment:
added in a required field for nc d400 xml and pdf to make d400 xml stop erroring out.